src/gitlabci_lint/validate.py

```diff
@@ -86,13 +86,13 @@ def validateCiConfig(token: str, baseUrl: str, configs: List[str], silent: bool)
                     with urlopen(request) as response:
                         lint_output = json.loads(response.read())
 
-                    if lint_output['status'] == 'invalid':
+                    if not lint_output['valid']:
```

Comment:
Is this field `lint_output['valid']` now a boolean?

Reply:
Yeah, they changed the response model [here](https://docs.gitlab.com/ee/api/lint.html)